float64 simTime
rdb_object_state_base base
rdb_object_state_ext ext